omg/entities/player.py

```diff
@@ -107,9 +108,9 @@ def update(self, mouse_x, mouse_y, delta_time):
         self._regenerate_mana(delta_time)
 
         # Update combined elements
-        if len(self.to_be_combined_element_cash) == 2:
+        if len(self.to_be_combined_element_buffer) == 2:
             # Get the name of the skill after combining elements
-            new_skill = "".join(self.to_be_combined_element_cash[::])
+            new_skill = "".join(self.to_be_combined_element_buffer[::])
 
             # Remove "ElementFactory" from the element name
             new_skill = new_skill.replace("ElementFactory", "")
```

Comment:
This logic depending on the ElementFactory class name still stands, as far as I understand. This should be removed as it does not offer a robust solution.